packages/office-ui-fabric-react/src/components/Layer/Layer.tsx

```diff
@@ -1,13 +1,17 @@
+import { createPortal } from 'react-dom';
 import { styled } from '../../Utilities';
 import {
   ILayerProps,
   ILayerStyleProps,
   ILayerStyles
 } from './Layer.types';
 import { LayerBase } from './Layer.base';
+import { PortalLayerBase } from './PortalLayer.base';
 import { getStyles } from './Layer.styles';
 
+const portalSupport: boolean = !!createPortal;
```

Comment:
Very interesting... Just wondering out loud - would this cause problems for a consumer of OUFR 5.x using the ES2015 module imports?

Reply:
So, I think (and I might be wrong here), if a consumer is on React 15 and consumes the ES2015 exports (`/lib-es2015/`), this would be an invalid import due to the static nature of ES2015 modules... This would then break all usages of Fabric, for instance, if it was in the browser.